LCDd.conf

```diff
@@ -356,6 +356,9 @@ keypad_test_mode=no
 
 ## g15 driver for Logitech G15 Keyboard LCDs ##
 [g15]
+# Set this to yes to directly access the keyboard through a /dev/hidraw#
+# device instead of connecting to g15daemon
+#HidrawAccess=no
```

Comment:
Is there any way to have this auto-detected? E.g. is it save to assume that if we can connect to the g15daemon then we have to use it, because it did detach the kernel driver?

Reply:
That is a good idea, this also combines nicely with making libg15daemon_client optional now that we also support direct access. So I've implemented both.